anipy_cli/cli.py

```diff
@@ -606,63 +633,413 @@ def binge(ep_list, quality, player):
+                self.binge_latest()
+            elif picked == "q":
+                self.quit()
+            elif picked == "m":
+                self.create_gogo_maps()
+            else:
+                error("invalid input")
+            self.print_opts()
+
+    def add_anime(self):
+        show_entry = entry()
+        searches = []
+        if (
+            not self.no_season_search
+            and input("Search for anime in Season? (y|n): \n>> ") == "y"
+        ):
+            searches = get_season_searches(gogo=False)
+
+        else:
+            searches.append(input("Search: "))
+
+        #
+        for search in searches:
+            if isinstance(search, dict):
+                mal_entry = search
+
+            else:
+                print("\nCurrent: ", search)
+                temp_search = self.m_class.get_anime(search)
+                names = [item["node"]["title"] for item in temp_search]
+                print_names(names)
+                selected = False
+                skip = False
+                while not selected:
+                    try:
+                        sel_index = input("Select show (Number or c for cancel):\n")
+                        if sel_index == "c":
+                            skip = True
+                            break
+                        selected = int(sel_index)
+                    except ValueError:
+                        print("Please enter a valid number.")
+                if skip:
+                    continue
+                mal_entry = temp_search[selected]
+
+            self.m_class.update_anime_list(
+                mal_entry["node"]["id"], {"status": "watching"}
+            )
+            self.create_gogo_maps()
+
+        clear_console()
+
+    def del_anime(self):
+        mal_list = self.m_class.get_anime_list()
+        mal_list = [x for x in mal_list]
+        mal_names = [n["node"]["title"] for n in mal_list]
+        print_names(mal_names)
+        while True:
+            inp = cinput("Enter Number: ", colors.CYAN)
+            try:
+                picked = mal_list[int(inp) - 1]["node"]["id"]
+                break
+            except:
+                error("Invalid Input")
+
+        self.m_class.del_show(picked)
+        clear_console()
+
+    def list_animes(self):
+        for i in self.m_class.get_anime_list():
+            status = i["node"]["my_list_status"]["status"]
+
+            if status == "completed":
+                color = colors.MAGENTA
+            elif status == "on_hold":
+                color = colors.YELLOW
+            elif status == "plan_to_watch":
+                color = colors.BLUE
+            elif status == "dropped":
+                color = colors.ERROR
+            else:
+                color = colors.GREEN
+
+            print(
+                "==> Last watched EP: {} | ({}) | {}".format(
+                    i["node"]["my_list_status"]["num_episodes_watched"],
+                    color(status, color),
```

Comment:
wrong function name here, should be cprint

Reply:
Print is being used in several other locations as well as input. Let me fix dis real quick

Reply:
while you are at it could you change the ints in the print statement  [here](https://github.com/sdaqo/anipy-cli/blob/master/anipy_cli/download.py#L343) to strings?

Reply:
sure ^^